packages/valory/contracts/registries_manager/contract.py

```diff
@@ -83,11 +83,18 @@ def create(  # pylint: disable=too-many-arguments
         contract_interface = cls.get_instance(
             ledger_api=ledger_api, contract_address=contract_address
         )
-        tx_hash = contract_interface.functions.create(
+        tx_params = {
+            "from": owner,
+            "nonce": ledger_api.api.eth.get_transaction_count(
```

Comment:
`ledger_api.api.eth.get_transaction_count`

looks like `get_transaction_count` has to be exposed on the LedgerApi level?

Reply:
It might be let me check

Reply:
i understand.
this solution is  LedgerApi implementation based (ethereum only)
probably add LedgerApi.identifier check? and raise ValueError with LEdgerApi only ethereum supported?

Reply:
Let's open an issue for this

Reply:
https://github.com/valory-xyz/open-aea/blob/main/plugins/aea-ledger-ethereum/aea_ledger_ethereum/ethereum.py#L1421

Reply:
As @solarw you are re-implementing the wheel